tests/libs/datasets/ca_vaccination_backfill_test.py

```diff
@@ -0,0 +1,70 @@
+from tests import test_helpers
+import pytest
+from libs.pipeline import Region
+from covidactnow.datapublic.common_fields import CommonFields
+
+from libs.datasets import ca_vaccination_backfill
+
+
+def test_ca_county_vaccination_assertion():
+
+    region_ca = Region.from_state("CA")
+    region_la = Region.from_fips("06037")
+    region_mo = Region.from_state("MO")
+
+    input_data = {
+        region_ca: {
+            CommonFields.VACCINES_ADMINISTERED: [100, 200],
+            CommonFields.VACCINATIONS_INITIATED: [50, 150],
+            CommonFields.VACCINATIONS_COMPLETED: [50, 50],
+        },
+        region_la: {
+            CommonFields.VACCINES_ADMINISTERED: [100, 200],
+            CommonFields.VACCINATIONS_INITIATED: [50, 150],
+        },
+        region_mo: {CommonFields.VACCINATIONS_COMPLETED_PCT: [25.0, 28.0]},
+    }
+    static_by_region_then_field_name = {region_la: {CommonFields.POPULATION: 1000}}
+    dataset = test_helpers.build_dataset(
+        input_data, static_by_region_then_field_name=static_by_region_then_field_name
+    )
+
+    with pytest.raises(AssertionError):
+        dataset_out = ca_vaccination_backfill.derive_ca_county_vaccine_pct(dataset)
+
+
+def test_ca_county_vaccination_calculation():
+
+    region_ca = Region.from_state("CA")
+    region_la = Region.from_fips("06037")
+    region_mo = Region.from_state("MO")
+
+    input_data = {
+        region_ca: {
+            CommonFields.VACCINES_ADMINISTERED: [100, 200],
+            CommonFields.VACCINATIONS_INITIATED: [50, 150],
+            CommonFields.VACCINATIONS_COMPLETED: [50, 50],
+        },
+        region_la: {CommonFields.VACCINES_ADMINISTERED: [100, 200]},
+        region_mo: {CommonFields.VACCINATIONS_COMPLETED_PCT: [25.0, 28.0]},
+    }
+    static_by_region_then_field_name = {region_la: {CommonFields.POPULATION: 1000}}
+    dataset = test_helpers.build_dataset(
+        input_data, static_by_region_then_field_name=static_by_region_then_field_name
```

Comment:
how about adding a function so you can write
```
static_by_region_then_field_name = {region_la: {CommonFields.POPULATION: 1000}, **test_helpers.make_static_by_region_fields(input_data.keys())}
```
then leave `test_helpers.build_dataset` unmodified?